src/core/sys/dragonflybsd/time.d

```diff
@@ -0,0 +1,32 @@
+/**
+ * D header file for DragonFlyBSD's extensions to POSIX's time.h.
+ *
+ * Copyright: Copyright 2014
+ * License: $(WEB www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+ * Authors: Jonathan M Davis,Diederik de Groot(port:DragonFlyBSD)
+ * Copied:  From core/sys/freebsd/sys
+ */
+module core.sys.dragonflybsd.time;
+
+version(DragonFlyBSD):
+
+public import core.sys.posix.time;
+
+extern (C):
```

Comment:
Enums are like preprocessor defines in C/C++, they're pasted in and have no linkage, so this does nothing.

Reply:
Remarked out 'extern(C):'